examples/oneview_scope.yml

```diff
@@ -1,5 +1,5 @@
 ###
-# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+# Copyright (2020) Hewlett Packard Enterprise Development LP
```

Comment:
if you have no changes, please don't change copyright year.

Reply:
Updated as before